src/cellflow/metrics/_metrics.py

```diff
@@ -21,12 +23,38 @@
 
 
 def compute_r_squared(x: ArrayLike, y: ArrayLike) -> float:
-    """Compute the R squared between true (x) and predicted (y)"""
+    """Compute the R squared score between means of the true (x) and predicted (y) distributions.
+
+    Parameters
+    ----------
+        x
+            An array of shape [num_samples, num_features].
+        y
+            An array of shape [num_samples, num_features].
+
+    Returns
+    -------
+        A scalar denoting the R squared score.
+    """
     return r2_score(np.mean(x, axis=0), np.mean(y, axis=0))
 
 
 def compute_sinkhorn_div(x: ArrayLike, y: ArrayLike, epsilon: float = 1e-2) -> float:
-    """Compute the Sinkhorn divergence between x and y."""
+    """Compute the Sinkhorn divergence between x and y as in Feydy et al. :cite:`feydy:19`.
```

Comment:
Just cite once, i.e. `Compute the Sinkhorn divergence between x and y as in  :cite:`feydy:19`.
Similarly in other places, please.